MIP61/MIP61.md

```diff
@@ -112,13 +120,13 @@ This component addresses the situation when a Recognized Delegate intends to mig
 
 Suppose the parameters are set to their initial values.
 
-With 100 MKR delegated, the compensation is calculated as `C*min(1,100^q/10000^q)`. This is `C*10/100` = 14 400 DAI per year.
+With 100 MKR delegated, 95% Participation, and 95% Communication, the compensation is calculated as `min(max(min(95%,95%)-75%,0)/15%,1) C*min(1,100^q/10000^q)`. This is `100% C*10/100` = 14 400 DAI per year.
```

Comment:
Can we try to format these in a way that doesn't make readers want to off themselves? :D

Would suggest calculating each major parameter on separate lines, and then combining at the end.

Reply:
Also provide a clear key as to what letter is what parameter.

Reply:
How about using [clamp](https://en.cppreference.com/w/cpp/algorithm/clamp) instead of the `min(max(...))` mess?

Reply:
Yep, try that and see if it makes it more readable.